packages/forms-web-app/src/views/manage-appeals/add-remove-users.njk

```diff
@@ -0,0 +1,29 @@
+{% extends "layouts/lpa-dashboard/main.njk" %}
+{% from "govuk/components/back-link/macro.njk" import govukBackLink %}
+{% from "govuk/components/button/macro.njk" import govukButton %}
+
+{% block backButton %}
+    {{ govukBackLink({
+        text: "Back to appeals",
+        href: dashboardUrl,
```

Comment:
have coded this in rather than using the backlink in main (which, I think, uses the navigation middleware), as the link seems to indicate it should always direct back to the appeals dashboard